Rogue-Ape-Studios-XR/Secrets-of-Ignacios
# 🍱 Added Air visual effects

## Content

Visual effects for a wind projectile, hold effect and impact have been added to the project.

## Changes

### Added
`Air materials, shaders, models and vfx graphs` : Were created.


## Testing

The feature / Bugfix can be testing by following these steps:

1. Open scene
2. Place VFX, it should look like this.

![SOIairVFX](https://github.com/user-attachments/assets/1f492d1f-184c-41ff-b8da-57880e9fdd1c)
